cluster/images/etcd/attachlease.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"strings"
+	"time"
+
+	"github.com/coreos/etcd/clientv3"
+	"github.com/golang/glog"
+	"golang.org/x/net/context"
+)
+
+func main() {
+	var etcdAddress string
+	var ttlKeysPrefix string
+	var leaseDuration time.Duration
+	flag.StringVar(&etcdAddress, "etcd-address", "", "Etcd address")
+	flag.StringVar(&ttlKeysPrefix, "ttl-keys-prefix", "", "Prefix for TTL keys")
+	flag.DurationVar(&leaseDuration, "lease-duration", time.Hour, "Lease duration")
```

Comment:
That's not true - for flag duration you can pass e.g. 10m or 1h or 30s or even 500ms - it actually doesn't have seconds granularity.